politeiawww/cmd/politeiavoter/politeiavoter.go

```diff
@@ -570,70 +573,99 @@ func (c *ctx) eligibleVotes(vrr *v1.VoteResultsReply, ctres *pb.CommittedTickets
 	return eligible, nil
 }
 
-func (c *ctx) _inventory() (*v1.ActiveVoteReply, error) {
-	responseBody, err := c.makeRequest(http.MethodGet, v1.RouteActiveVote, nil)
+func (c *ctx) _inventory(page uint32) (*tkv1.InventoryReply, error) {
+	i := tkv1.Inventory{
+		Page:   page,
+		Status: tkv1.VoteStatusStarted,
```

Comment:
This is too specific.  When you create a function for an API call like this is should be generic, i.e. allow the caller to provide the status instead of hard coding in `VoteStatusStarted`, so that it can be re-used if new functionality needs to be added in the future.

Reply:
👍🏽